manifests/init.pp

```diff
@@ -35,10 +35,13 @@
         # place the key in the keyrings directory where apt won't search for keys for all repos
         # ascii encoded files need to end with *.asc, binary files with .gpg...
         file { '/usr/share/keyrings/lldpd.asc':
-          source => "https://download.opensuse.org/repositories/home:/vbernat/${repourl}/Release.key",
-          owner  => 'root',
-          group  => 'root',
-          mode   => '0644',
+          ensure         => 'file',
+          source         => "https://download.opensuse.org/repositories/home:/vbernat/${repourl}/Release.key",
+          owner          => 'root',
+          group          => 'root',
+          mode           => '0644',
+          checksum_value => '2e532e3f800b788b0248da86b1cd722e58e9c99413912fd029c20d88d55ebadc',
```

Comment:
maybe we should put this into hiera? like the gpgfingerprint (which is not used in this file)

Reply:
I moved it to a parameter. `$gpgfingerprint` was used in the past, but removing it requires a major release.